packages/components/atoms/f-button/test/specs/accessibility/axe-accessibility.spec.js

```diff
@@ -0,0 +1,15 @@
+import { getAccessibilityTestResults } from '../../../../../../../test/utils/axe-helper';
+import ButtonComponent from '../../../test-utils/component-objects/f-button.component';
+
+describe('Accessibility tests', () => {
+    beforeEach(() => {
+        browser.url('?path=/story/components-atoms--button-component');
+        browser.switchToFrame(0);
+        ButtonComponent.waitForButtonComponent();
+    });
+
+    it('a11y - should test f-button component WCAG compliance', () => {
+        // Act
+        const axeResults = getAccessibilityTestResults('f-button');
```

Comment:
Shouldn't these contain `expect` calls here to check the results returned by the helper?

Reply:
Yep! As mentioned in the description of this ticket, for the time being, it's just giving us visibility of these violations. Currently, this is in the form of a CSV produced by Circle, as seen here - https://app.circleci.com/pipelines/github/justeat/fozzie-components/3592/workflows/e631d049-1757-46ca-b2cd-8d324a18d570/jobs/4010/artifacts


Once this ticket gets merged, I'll do the following:

- Create a ticket to fix any a11y violations
- Create a ticket to add asserts to these tests and make them a blocking step